software_steering_council.md

```diff
@@ -0,0 +1,52 @@
+# Software Steering Council
+
+## Purpose
+
+The Jupyter Software Steering Council (SSC) has jurisdiction over software-related decisions across Project Jupyter, with a primary focus on coordination across projects and decisions that have impact across many Jupyter Subprojects. It is also a mechanism for representatives of each project to share information and expertise. Technical decisions and processes where the SSC isn't explicitly involved are automatically delegated to the individual projects to manage their day-to-day activities, create new repositories in their orgs, etc., with independence and autonomy.
+
+## Responsibilities
+
+### Sole responsibilities
+
+The SSC is responsible for the following:
+
+- Defining the submission, review, and approval process for Jupyter Enhancement Proposals.
+- Stewarding the JEP process to ensure that it is inclusive and participatory, and solicits feedback from the right stakeholders. As a starting point, [the NumFOCUS DISCOVER Cookbook](https://github.com/numfocus/DISCOVER-Cookbook) may offer some useful starting pointers.
+- Making decisions on JEPs after receiving community feedback (JEPs are the mechanism for elevating any software decision/idea up to the broader Project, while individual Subprojects can make decisions within their scope via their regular mechanisms).
+- Owning, managing and implementing the Jupyter Incubation process (https://github.com/jupyter-incubator).
+- Owning, managing and implementing the Jupyter Attic process (https://github.com/jupyter-attic).
+- Stewarding discussions and activities beyond JEPs that involve cross-cutting concerns, standards, protocols and other architectural issues that impact the entire project.
+- Owning and managing anything related to security vulnerabilities across the project, including the Jupyter security lists, any private security repositories, etc.
```

Comment:
It feels like this might be delegated to a sub-group to ensure that there is more direct accountability to follow up in security issues. You don’t want to dilute accountability if there is a major vulnerability that needs to be fixed ASAP

Reply:
Good point - we'll clarify it by naming a working group that captures the work that's been already going on (mailing lists, response, CVE management, etc). But the main point here is to capture the _ownership and responsibility_ of security issues ultimately rests with the SSC. It can operationalize that by delegation to a working group, but at the end of the day it's responsible for handling those issues across Jupyter.

Let us know if that isn't clear enough :)